### PR TITLE
Added Mate HW5.3 Solution

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +13,31 @@
...
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String text = null;
+        try {
+            text = Files.readString(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        if (text.length() == 0) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to create a solution without this check

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +13,31 @@
...
+            return new String[0];
+        }
+        String[] words = text.toLowerCase().replaceAll("[^a-z]+", " ").split(" ");
+        StringBuilder newContainer = new StringBuilder();
+        for (String word : words) {
+            if (word.charAt(0) == LETTER) {
+                newContainer.append(word + " ");
+            }
+        }
+        if (newContainer.length() == 0) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to create a solution without this check, too

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/FileWork.java]: Fixed.

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/main/java/core/basesyntax/FileWork.java]: Fixed.

**Pull request closed without merging**